dll/win32/shell32/CUserNotification.cpp

```diff
@@ -76,7 +78,7 @@ VOID CUserNotification::RemoveIcon()
 {
     NOTIFYICONDATAW nid = {0};
 
-    nid.cbSize = NOTIFYICONDATAW_V3_SIZE; // sizeof(nid);
+    nid.cbSize = NOTIFYICONDATAW_V3_SIZE;
```

Comment:
please add a static_assert here that `sizeof(nid) >= NOTIFYICONDATAW_V3_SIZE`.

Reply:
Inserted.